### PR TITLE
Release: @slack/webhook@7.0.4, @slack/web-api@7.8.0

### PR DESCRIPTION
**Pull request by hello-ashleyintech**
### Summary

This PR releases the following new versions:
- @slack/webhook@7.0.4
- @slack/web-api@7.8.0

### Requirements (place an `x` in each `[ ]`)

* [x] I've read and understood the [Contributing Guidelines](https://github.com/slackapi/node-slack-sdk/blob/main/.github/contributing.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).

### CODE REVIEW

File: packages/webhook/package.json
Comment (zimeg) [inline packages/webhook/package.json:3]: We might be alright to keep this as a patch bump after landing [the dependency bump](https://github.com/slackapi/node-slack-sdk/milestone/69?closed=1)?

**Review by zimeg — COMMENTED** [review]
Quick question before merging!

File: packages/webhook/package.json
Comment (zimeg) [reply packages/webhook/package.json:3]: I often put package updates as patch changes but please let me know if `semver:minor` is the way here 🙏

Comment (hello-ashleyintech) [reply packages/webhook/package.json:3]: This one is a bit weird because it's a dependency bump, but also supposed to also resolve a security vuln 🤔 But I guess that vuln is considered a bug, so perhaps patch makes more sense here?

Comment (hello-ashleyintech) [reply packages/webhook/package.json:3]: I have pondered this and agree that patch makes more sense, especially when thinking in terms of dependabot PRs. I'll update the webhook milestone!

Comment (zimeg) [reply packages/webhook/package.json]: ✨ 🐛 🔍  I like to think a `security` vuln. makes it a fancy patch?

I was thinking about @dependabot too and was also curious how updates have been in past minor vs. patch changes, but [`@slack/webhook`](https://www.npmjs.com/package/@slack/webhook?activeTab=versions) doesn't hint at much here!

**Review by zimeg — APPROVED** [review]
Thanks for the fast follows and seeing these updates through! 🙏

**Comment by zimeg** [comment]
The `@slack/oauth` package just popped into mind which [uses](https://github.com/slackapi/node-slack-sdk/blob/96d846a79878f0d33893d14a9942ed9c4f678f2d/packages/oauth/package.json#L35) `@slack/web-api` - bumping `@slack/web-api` might be needed too? 😓

Not sure if this was planned for a separate PR, but I think it could be changed here too. Though separate PRs feels a bit more stable? Your call!

**Comment by hello-ashleyintech** [comment]
@zimeg for public transparency will also leave this comment here- I will merge this PR, release `web-api` and `webhooks`, and then on Monday I'm going to dig into updating those dependencies and releasing - so for Bolt, `socket-mode`, and `oauth` (thus far)! I'll look into other places as well where we may need to update :raised_hands:

**Pull request merged**
